mlrun/utils/helpers.py

```diff
@@ -268,12 +268,17 @@ def validate_tag_name(
 def validate_artifact_key_name(
     artifact_key: str, field_name: str, raise_on_failure: bool = True
 ) -> bool:
+    if field_name == "artifact.key":
+        type = "key"
+    else:
+        type = "db_key"
```

Comment:
type is built in function

```suggestion
   field_type = "key" if field_name == "artifact.key" else "db_key"
```